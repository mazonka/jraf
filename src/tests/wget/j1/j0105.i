command=jr ping : version backend : version client : version blah