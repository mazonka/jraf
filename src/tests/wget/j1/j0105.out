OK : OK 10420 : JRAF_ERR no file system found [wroot/.jraf.sys/version]